ocd_backend/utils/file_parsing.py

```diff
@@ -33,12 +33,13 @@ def file_parser(fname, pages=None):
             # reraise everything
             raise
     else:
-        try:
-            content = parser.from_file(fname)['content']
-            return (content or '').encode('UTF-8')
-        except:
-            # reraise everything
-            raise
+        # try:
```

Comment:
When leaving commented code, please mention the issue in the commit or in a leading comment so others can discover why the behaviour was changed, 'it is fixed' is only helpful with prior domain knowledge.